include/fmt/compile.h

```diff
@@ -36,7 +36,7 @@ template <typename OutputIt> class truncating_iterator_base {
   using difference_type = std::ptrdiff_t;
   using pointer = void;
   using reference = void;
-  using _Unchecked_type =
+  using FMT_UNCHECKED_TYPE =
       truncating_iterator_base;  // Mark iterator as checked.
```

Comment:
This should be `FMT_UNCHECKED_TYPE(truncating_iterator_base);

Reply:
Doh, sorry I missed that one. Fixed.

Reply:
...Oh, wait, it's a different keyword...